bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -252,6 +263,11 @@ public boolean isCompletionMarker(Result[] result) {
     public long getRowCount(Result[] result) {
       return result == null ? 0 : result.length;
     }
+
+    @Override
+    public ByteString getKey(Result[] result) {
+      return isCompletionMarker(result) ? null : ByteString.copyFrom(result[result.length - 1].getRow());
```

Comment:
The CloudBigtableIO worked because of this code: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/master/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/BigtableResultScannerAdapter.java#L39

We should still have a result.length == 0 check in isCompletionMarker().  I'll add it.